src/analyzer.ts

```diff
@@ -198,15 +214,17 @@ export class BuildAnalyzer {
    * Return _dependenciesOutputStream, which will contain fully loaded file
    * objects for each dependency after analysis.
    */
-  get dependencies(): NodeJS.ReadableStream {
+  dependencies(): NodeJS.ReadableStream {
```

Comment:
startAnalysis() begins the loading of files from disk. This was a significant enough side effect to warrant the change. And as a little bonus, it also now matches the `polymerProject.dependencies()` method that most users interact with.